backend/prisma/schema.prisma

```diff
@@ -8,11 +8,11 @@ datasource db {
 }
 
 model User {
-  id   String @id @default(uuid())
-  email   String   @unique
-  firstName    String 
-  lastName String 
-  password String 
-  birthDate DateTime? 
+  id        String    @id @default(uuid())
+  email     String?    @unique
```

Comment:
Do you know if we will run into problems in our application if we don't have an email provided for a user? 

When we thought about local strategy, it would require everyone to have an email associated w/ weWrap.

Is there a way to pull their email and store it in our database from the oauth response?

Reply:
Yup, I figured out how to add the email scope when we request user data. I will change our email attribute back to non-nullable.